### DISCUSSION TITLE
GPU out of memory when generating larger textures

### DISCUSSION THREAD

**Discussion by EvanDaley** [carson-katri/dream-textures#628] [General]
If I try to increase the resolution at all, I run into "CUDA out of memory". Are there any workarounds, aside from getting a better GPU? Can I run it in multiple steps or over a longer period of time? 

I did experiment with upscaling the 512x image, but didn't like that workflow, and I think there would be a lot of value, generally, in being able to render the higher res textures on the first run. 

Advice would be greatly appreciated - thank you!

Also, this package is amazing. Really great work!

**Comment by carson-katri** [comment]
A new attention mechanism was added in #575 and will be available in the next version. It reduces VRAM consumption quite a bit and may let you generate larger images.
